docs/source/overview/options.md

```diff
@@ -119,13 +119,13 @@ find more details in [Olive Models](https://microsoft.github.io/Olive/api/models
           - If it is `False`, Olive will not use key value cache.
           - If it is `True`, Olive will infer the cache configuration from the input_names/input_shapes and input model based on default `kv_cache`.
           - If it is a dictionary, it should contains the key value cache configuration. Here is an default configuration example:
-            - `ort_past_key_name`: "past_key_<id>"
+            - `ort_past_key_name`: "past_key_values.<id>.key"
```

Comment:
If the purpose to update io_config names is to provide a simpliest example for llama2, then I am ok with this changes.

But in general, the original value looks clearer and simpler.

Reply:
The reason is not for a simpler example. But like I mentioned in the PR description, this template is the more common template. It is used by both genai and optimum.

I have only seen the previous template in the ort phi2 example.

Reply:
This makes the model directly compatible with both generate apis. Otherwise, we have to edit the genai config for genai api. It probably doesn't work for optimum.

Reply:
Ic. Sorry I missed the PR description.
The changes look good to me.